### ISSUE TITLE
tests/gold/html/Makefile uses bashism

### ISSUE THREAD

**Issue by rillig** [coveragepy/coveragepy#983]
**Describe the bug**
The `[[` keyword only exists in bash, not in other shells.
It should be replaced with a simple `[`, and the filename should be enclosed in double quotes.

**Comment by pawamoy** [comment]
It exists in zsh as well.

**Comment by nedbat** [comment]
Would you like to make a pull request?

**Comment by rillig** [comment]
No, I don't like to fork a whole repository, make a small change and then delete the repository again just to change a single line. Here's the patch:

~~~patch
--- tests/gold/html/Makefile.orig	2020-01-11 21:34:28.000000000 +0000
+++ tests/gold/html/Makefile	2020-05-02 14:46:46.837476331 +0000
@@ -7,7 +7,7 @@
 
 complete: 	## Copy support files into directories so the HTML can be viewed properly.
 	@for sub in *; do \
-		if [[ -f $$sub/index.html ]]; then \
+		if [ -f "$$sub/index.html" ]; then \
 			echo Copying into $$sub ; \
 			cp -n support/* $$sub ; \
 		fi ; \
~~~

**Comment by pawamoy** [comment]
It definitely exists in other shells: `zsh` and `busybox ash` have it. But `dash` doesn't indeed.

About forks, you can edit a file directly inside the coverage repo: GitHub will automatically fork it, create a `patch-1` branch and open the GitHub editor. Once satisfied you click on commit and it creates a PR. You'd still have to delete the fork from your repositories afterwards though.

**Comment by rillig** [comment]
See #984; sorry, I forgot to refer back to this issue in the commit.

**Comment by nedbat** [comment]
Fixed in #984.
